core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/model/SqlBuilder.java

```diff
@@ -138,4 +142,21 @@ public void setTail(String tail, Long... placeHolders) {
             count++;
         }
     }
+
+    public String getSql() {
```

Comment:
Please add JavaDoc for the public methods

Reply:
addressed with https://github.com/wso2/carbon-kernel/pull/2303